.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -404,16 +407,30 @@ kernel_matrix_testing_cleanup:
   script:
     - |
       if [ -f $CI_PROJECT_DIR/stack.outputs ]; then
-        INSTANCE_IPS=$(cat $CI_PROJECT_DIR/stack.outputs | grep -e '-instance-ip' | cut -d ' ' -f 2 | tr '\n' ' ' | awk '{$1=$1};1' | tr ' ' ',')
+        INSTANCE_IPS=$(cat $CI_PROJECT_DIR/stack.outputs | grep -e "$ARCH-instance-ip" | cut -d ' ' -f 2 | tr '\n' ' ' | awk '{$1=$1};1' | tr ' ' ',')
```

Comment:
We only do `grep $ARCH-instance-ip | cut -d ' ' -f 2` in the run_tests jobs, is that sufficient here?

Reply:
also, please change to `INSTANCE_IP` since it is singular now